src/backend/crossterm.rs

```diff
@@ -151,7 +129,7 @@ impl Backend for CrosstermBackend {
             }
             s.object_style.attrs = cell.style.modifier.into();
 
-            self.crossterm.paint(s).map_err(convert_error)?;
+           write!(io::stdout(), "{}", s);
```

Comment:
Isn't this locking stdout every time you write ? Shouldn't you lock once and write to `StdoutLock` ?

Reply:
I can try that, notice that that happened before too.

Reply:
I could misunderstand the [documentation](https://doc.rust-lang.org/std/io/fn.stdout.html) but for me locking outside the for loop would avoid the implicit synchronization. Furthermore, the struct returned by `io::stdout().lock()` is also writable.

Reply:
Right, I will take a look at if this could be done more efficiently.

Reply:
I moved the lock out of the loop, by doing this we only lock outside the loop, this lock is then used in the loop for drawing all cells.